examples/nlp/albert_squad_pytorch/startup-hook.sh

```diff
@@ -2,3 +2,4 @@
 pip install sentencepiece==0.1.91
 pip install transformers==3.1.0
 pip install -e git+git://github.com/LiyuanLucasLiu/RAdam.git@baf4f65445c00d686d4098841b3ca1f62a886326#egg=radam
+pip install pynvml
```

Comment:
This is a reminder that we now rely on this library and need to add this to the environments.